README.md

```diff
@@ -38,114 +46,106 @@ The league establishes a map pool for each season. The map pool for Season 1 of
 - Towers of Doom
 - Warhead Junction
 
-[View Map Pool Here](https://www.chairleague.com/maps/enabled)
-
-Study up, hone your strats, and be ready to throw down on these battlefields on your preferred night of Chair League action.
+[View Map Pool Here](https://www.chairleague.com/maps/enabled).
 
-### Substitutions
-
-Matches require four or five players from your team to check in before each round can start. Teams may sub in one non-rostered player, so long as that player is eligible to play in that league (see [Player on Multiple Teams](#player-on-multiple-teams) and [Selecting a Division](#selecting-a-division)).
+### Hero Status
 
-Substitutions can occur in between rounds. A player who plays in any round is not required to play in any other rounds in the series. 
+Generally, only heroes with significant bugs are banned, at Chair League's discretion. Heroes are available upon release.
```

Comment:
"Significant and exploitable"  (due to the nazeebo conversation from this week)

Also "it is generally the players responsibility to be aware of bugs that detriment an individual characters performance."